docs.package.json

```diff
@@ -36,8 +36,8 @@
     "label": "testbed"
   },
   {
-    "repo": "osism/cloud-in-a-box",
-    "source": "doc",
+    "repo": "osism/docs",
+    "source": "advanced-guides/cloud-in-a-box.md",
```

Comment:
This is the path in the new repository: https://github.com/osism/osism.github.io/tree/main/docs/advanced-guides